packages/plan-form-core/src/formHelpers/constants/lang.ts

```diff
@@ -52,11 +52,6 @@ export const PRODUCT_CHECK_ACTIVITY_DESCRIPTION =
   'Check for all products (100%) within the jurisdiction';
 export const PRODUCT_CHECK_GOAL_MEASURE = 'Percent of products checked';
 
-export const FIX_PRODUCT_PROBLEM_ACTIVITY = 'Fix Problem';
```

Comment:
@ciremusyoka removed these, I think they are the only ones that became surplus,

Reply:
there isn't any other language file, this module does not have translation json files.